src/main/java/ua/lviv/iot/parkingServer/logic/exception/message/ErrorMessageModel.kt

```diff
@@ -0,0 +1,8 @@
+package ua.lviv.iot.parkingServer.logic.exception.message
+
+class ErrorMessageModel(
+    var status: Int? = null,
+    var message: String? = null
+) {
+
+}
```

Comment:
please remove `{ }`

Can we change `Int?` to `Int`? What about `message` type nullability?